modules/replicator/src/main/java/org/apache/ignite/internal/replicator/Replica.java

```diff
@@ -73,4 +74,11 @@ public interface Replica {
      * Shutdowns the replica.
      */
     CompletableFuture<Void> shutdown();
+
+    /**
+     * Updates peers and learners lists in raft client.
```

Comment:
Let's use the generic "replication client" instead of "raft", because we want to be more abstract about replication protocol.